packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -154,6 +155,7 @@ export type AzureCommunicationCallWithChatAdapterArgs = {
     credential: CommunicationTokenCredential;
     locator: CallAndChatLocator | TeamsMeetingLinkLocator;
     alternateCallerId?: string;
+    callingFeatures?: CallAdapterOptionalFeatures;
```

Comment:
nit: I would keep this consistent and make this features, when it does include chat features the type can be updated to `CallWithChatAdapterOptionalFeatures` and still be a non-breaking change